apps/styleguide/src/layout/LayoutModal.vue

```diff
@@ -4,7 +4,7 @@
       v-if="show"
       class="modal fade show"
       role="dialog"
-      style="display: block"
+      style="display: block; overflow: scroll"
```

Comment:
why use style instead of style tag and css, and why would this style be needed , what introduced this bug ? , just add more style weight might hide the underlying issue